### ISSUE TITLE
post products so others can download

### ISSUE THREAD

**Issue by balhoff** [NCATS-Tangerine/cam-pipeline#56]
E.g. Blazegraph journal, RDF dump, etc. (cc @deepakunni3)

**Comment by gaurav** [comment]
Should that be on stars.renci.org? Since all our inputs are open source, we could also consider sticking the TSV output file in Zenodo or something. I bet @EvanDietzMorris has the Neo4J files available on stars.renci.org, and we should provide links to those as well.

**Comment by EvanDietzMorris** [comment]
That's right, the /metadata for each graph on Automat has a link to the neo4j dump, and recently we decided to always include the jsonl nodes and edges files too, so that will show up in future builds. If we want the latest neo4j dump to always live at the same exact link we'll need to discuss and implement that (it might be nice in general to have a "latest" sort of thing set up).
https://automat.renci.org/cam-kp/metadata
https://stars.renci.org/var/plater/bl-3.5.4/CAMKP_Automat/ded8cfcdee598475/
